store/datastore.go

```diff
@@ -27,14 +27,16 @@ type Storer interface {
 	UpsertVersion(id string, versionDoc *models.Version) error
 	UpsertContact(id string, update interface{}) error
 
+	GetDimensions(datasetID, editionID, versionID string) (*models.DatasetDimensionResults, error)
+
 	GetInstances(filter string) (*models.InstanceResults, error)
 	GetInstance(id string) (*models.Instance, error)
 	UpdateInstance(id string, instance *models.Instance) error
 	AddInstance(instance *models.Instance) (*models.Instance, error)
 	AddEventToInstance(instanceID string, event *models.Event) error
-	AddDimensionToInstance(dimension *models.Dimension) error
+	AddDimensionToInstance(dimension *models.CachedDimension) error
```

Comment:
Order alphabetically for ease of finding function

Reply:
Done